frontend/apps/web/app/(mgmt)/[account]/connections/[id]/page.tsx

```diff
@@ -69,3 +78,39 @@ export default function ConnectionPage({ params }: PageProps) {
     </OverviewContainer>
   );
 }
+
+interface CloneConnectionProps {
+  connectionType?: string;
+  id?: string;
```

Comment:
these should be required if they are needed for the method to function.

Reply:
yeah these are required for the routing to work correctly.